be/src/storage/task/engine_publish_version_task.cpp

```diff
@@ -119,9 +119,9 @@ OLAPStatus EnginePublishVersionTask::finish() {
                 }
             }
             partition_related_tablet_infos.erase(tablet_info);
-            VLOG(1) << "publish version successfully on tablet. tablet=" << tablet->full_name()
-                    << ", transaction_id=" << transaction_id << ", version=" << version.first
-                    << ", res=" << publish_status;
+            LOG(INFO) << "publish version successfully on tablet. tablet=" << tablet->full_name()
```

Comment:
Why change this log level?

Reply:
I will reverse this log.
I just want to add the log in line 149